src/containers/HomeV2/RecommendationListCard.jsx

```diff
@@ -66,41 +66,38 @@ class RecommendationListCard extends React.PureComponent {
 
     // temporary detection for CodeCombat Activities,
     // to change the avatar, logo, and color
-    const isCodeCombat = (title==="CodeCombat Activities")
-      ? true
-      : false;
+    const isCodeCombat = title === "CodeCombat Activities";
 
     return (
-      <div style={{marginBottom : "24px"}}>
-      {/* either use marginBottom here or put a <br /> at parent */}
+      <div style={{ marginBottom: "24px" }}>
+        {/* either use marginBottom here or put a <br /> at parent */}
         <Card style={styles.card}>
-          {(RecomType === "python") &&
+          {RecomType === "python" && (
             <Fragment>
               {/* this is temporarily testing the GUI with CodeCombat type
                   feel free to refactor it to a proper solution */}
               <CardHeader
                 avatar={
-                  <Avatar aria-label="Recommendation"
-                    style={isCodeCombat
-                      ? styles.avatarCC
-                      : styles.avatarPy}
+                  <Avatar
+                    aria-label="Recommendation"
+                    style={isCodeCombat ? styles.avatarCC : styles.avatarPy}
```

Comment:
@Brainenjii avatars need to be specific for different activity types, Jest, Game etc